pkgs/tools/misc/capture/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, pkgs, slop, ffmpeg, fetchFromGitHub, makeWrapper}:
+
+stdenv.mkDerivation rec {
+  name = "capture-${version}";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "buhman";
+    repo = "capture";
+    rev  = "4be986f17462b8d520559429c74da6bf3a436259";
+    sha256 = "172y06vs993x5v78zwl81xma1gkvjq1ad9rvmf3a217fyxsz4nhh";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  patches = [ ./0001-eval-fix.patch ./0002-sane-defaults.patch ];
+
+  installPhase = ''
+  install -Dm755 src/capture.sh $out/bin/capture
+
+  patchShebangs $out/bin/capture
+  wrapProgram $out/bin/capture \
+    --prefix PATH : '${stdenv.lib.makeBinPath [ slop ffmpeg ]}'
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A no bullshit screen capture tool";
+    homepage = "https://github.com/buhman/capture";
+    maintainers = [ maintainers.ar1a ];
```

Comment:
Needs a license https://search.nix.gsc.io/?q=license%20%3D&i=nope&files=&repos= (same with the other package.)

Reply:
neither packages have licenses, and as far as i could tell the default license is unfree - which would be accurate here i think.

Reply:
Hmm... I'm not sure we can really package software which isn't licensed... Maybe you're right, though. Can you open issues on those repos and ask them to apply a license?

Reply:
ptpb/pb_cli#12 buhman/capture#6 

I thought we could package unfree software, for example discord or spotify

Reply:
For sure we can, but they have a license saying they're unfree... but unlicensed is much less clear about rights, right?

Reply:
im not sure, i would argue that they just forgot to license the code especially because its advertised https://ptpb.pw/#shell-functions here but obviously that wouldn't really hold up. is there anyone you can ping who knows more about this stuff?

Reply:
I thought the GitHub TOS changed to grant some basic licenses, but it turns out to be much more minimal than I expected:

> If you publish your source code in a public repository on GitHub, according to the Terms of Service, other GitHub users have the right to view and fork your repository within the GitHub site.

I don't know ... I think there isn't a strict policy around requiring a license for Nixpkgs inclusion, but it sure seems there should be one ;)

I won't block on this problem.

Reply:
should i explicitly set it to unfree for the meantime then or just leave it omitted